scripts/prepare-fvt.sh

```diff
@@ -139,28 +139,12 @@ services:
     - https://${FVT_GATEWAY_HOST}:${FVT_API_PORT}/
   homePageRelativeUrl:  # Home page is at the same URL
   routedServices:
-    - gatewayUrl: api/v1  # [api/ui/ws]/v{majorVersion}
-      serviceRelativeUrl: api/v1/datasets
+    - gatewayUrl: api/v2  # [api/ui/ws]/v{majorVersion}
```

Comment:
gateway url api/v1 will change as well? 
my impression was only serviceRelativeUrl changes to api/v2

Reply:
This is for api mappings, nothing to do with the ui, essentially prior to this PR we never had tests for mvs explorer that checked any functionality, just that the page loaded so this didn't cause a problem up until now.